pkg/kubelet/dockershim/network/hostport/hostport_manager.go

```diff
@@ -310,7 +310,7 @@ func (hm *hostportManager) openHostports(podPortMapping *PodPortMapping) (map[ho
 	if retErr != nil {
 		for hp, socket := range ports {
 			if err := socket.Close(); err != nil {
-				klog.Errorf("Cannot clean up hostport %d for pod %s: %v", hp.port, getPodFullName(podPortMapping), err)
+				klog.ErrorS(err, "Cannot clean up hostport for the pod", "podName", getPodFullName(podPortMapping), "port", hp.port)
```

Comment:
What value `getPodFullName` returns? If it's `namespace/name` then we should not use `podName` as key. Best would be to find a way to extract name and namespace, or use key like `podFullName` just to avoid passing wrong value.